packages/notices/src/store/actions.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import { uniqueId } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_CONTEXT } from './constants';
+
+/**
```

Comment:
I really like how you document code. Just wanted to emphasize it as I look at this PR 💯 
How do you align @param definitions?

Reply:
>How do you align @param definitions?

I try to follow the guidelines in the [Documentation Standard](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/) which in practice are to keep each fragment space-aligned (type, name, description) within each line-delimited grouping (param, return, etc).

Reply:
I see, 80 characters per line is difficult to achieve but makes sense 👍

Reply:
I forgot the 80 character thing was part of the standard. It's self-imposed elsewhere. It becomes ugly and tempting to bypass in cases like this, but I think it's a motivator to try to avoid complexity (e.g. in the naming of the type).